.github/workflows/release.yml

```diff
@@ -45,38 +48,90 @@ jobs:
           export CXX=clang++
           # TODO: making changelog shouldn't depend on taichi.
           # We currently install taichi to make changelog when building the wheel.
-          python misc/ci_setup.py ci
-          cd python
-          python build.py build
-          # Uninstall taichi to make sure we test the wheel.
-          pip uninstall taichi -y
+          python -m pip install --user numpy pybind11 gitpython pytest
+          python setup.py develop
 
-      - name: Install Wheel and Test
+      - name: Test
         run: |
-          TAICHI_REPO_DIR=`pwd`
-          export PATH=$TAICHI_REPO_DIR/taichi-llvm/bin/:$PATH
-          NUM_WHL=`ls dist/*.whl | wc -l`
-          if [ $NUM_WHL -ne 1 ]; then echo 'ERROR: created more than 1 whl.' && exit 1 ; fi
-          pip install dist/*.whl
```

Comment:
Oh sorry for the late review, we actually want to preserve this logic :P Since we want to test a taichi installed from our built wheel (python setup.py develop is local dev mode which might slightly differ from the content inside the wheel). So we need another PR to bring this logic back. 
In fact it's indeed better to unify all the CI to first build the wheel and test from there. (compared to testing in dev mode).